### PR TITLE
Linux fixes

### PR DESCRIPTION
**Pull request by khaledhosny**
A few fixes for building and running FontView on Linux. it builds and runs fine locally, but Travis is hopelessly outdated so left out for now.

### CODE REVIEW

**Comment by brawer** [comment]
Cool, thanks a lot!

File: README.md [hunk 1]
```diff
@@ -17,10 +17,23 @@ porting should actually be quite easy). Likewise, please don’t
...
 
 ```sh
 $ git clone --recursive https://github.com/googlei18n/fontview.git
 $ cd fontview
 $ python2.7 build.py && ./build/FontView.app/Contents/MacOS/fontview
 ```
+
+## Building on Linux
+
+You need to first install wxWidgets as well as latest versions of FreeType,
```
Comment (HinTak) [inline README.md:30]: Ubuntu/Debian call their libwxbase3* and libwxgtk3*, while Fedora/Suse with mixed cases wx-base* and wxGTK3* . Might as well refer to the packages?

File: README.md [hunk 2]
```diff
@@ -17,10 +17,23 @@ porting should actually be quite easy). Likewise, please don’t
...
+HarfBuzz and FriBiDi.
+
+```sh
+$ git clone https://github.com/googlei18n/fontview.git
+$ cd fontview
+$ git submodule init
+$ git submodule update src/third_party/raqm/libraqm
```
Comment (HinTak) [inline README.md:37]: I do this in one step - 

```
git submodule update --init src/third_party/raqm/libraqm
```

File: build.py [hunk 3]
```diff
@@ -44,33 +44,43 @@ def build_mac(release):
         return False
     return True
 
+def check_pkgconfig(lib, min_version):
+    try:
+        subprocess.check_output(
+            ['pkg-config', '--exists', '%s >= %s' % (lib, min_version)])
```
Comment (HinTak) [inline build.py:50]: Do you mean `pkg-config %s --atleast-version=%s`? `--exists` seem to be wrong.19.0.13 is 2.7.1 or 2.7.0? I have 2.8 and it says 20.0.14. Harfbuzz only needs to be 1.4.0, I think?

Comment (HinTak) [reply build.py:50]: I was building against harfbuzz 1.3.2 (as shipped by fedora) and it seems to work.

Comment (khaledhosny) [reply build.py:50]: I used the versions corresponding to the submodules.

Comment (HinTak) [reply build.py:50]: Are you sure it is `--exists` rather than `--atheist-version=` ?

**Pull request merged**